code/utility/src/palindrome/palindrome_check/palindrome_check.py

```diff
@@ -0,0 +1,18 @@
+def isPalindrome(str):
+ 
+    # Run loop from 0 to len/2 
+    for i in range(0, len(str) // 2): 
+        if str[i] != str[len(str) - i - 1]:
```

Comment:
I'm pretty sure there is a better way to write `str[len(str) - i - 1]`, probably using negative indices.